scripts/backtest_uk_gsp.py

```diff
@@ -173,17 +169,14 @@ def get_available_t0_times(start_datetime, end_datetime, config_path):
     datapipes_dict["gsp"] = IterableWrapper([ds_fake_gsp])
 
     # Use create_t0_and_loc_datapipes to get datapipe of init-times
```

Comment:
A nit but I guess this comment is now slightly out of date?

Reply:
Yep, thank you!